content/news/2018-07-15-mapstruct-1_3_0_Beta1-is-out-with-builder-support.md

```diff
@@ -0,0 +1,186 @@
+---
+title: "Support for builders, mapper constructor injection and much more: MapStruct 1.3.0.Beta1 is out"
+author: Filip Hrisafov
+date: "2018-07-15"
+tags: [release, news]
+---
+
+It' my pleasure to announce the first Beta release of MapStruct 1.3.
+
+The new release comes with a whole lot of new functionality, e.g.:
+
+* Mapping of (immutable) objects using builders
+* Constructor injection for Annotation Based component models
+* `unmappedSourcePolicy` support
+* Allow defaultValue to be an expression
```

Comment:
Maybe add `(defaultExpression)` so that the users directly have a hint which property is responsible for this? Otherwise they might try to use `defaultValue` for this.